src/cephadm/cephadm

```diff
@@ -6861,6 +6929,18 @@ def _get_parser():
         help='daemon identifer for the exporter')
     parser_exporter.set_defaults(func=command_exporter)
 
+    parser_maintenance = subparsers.add_parser(
+        'host-maintenance', help='Manage the maintenance state of a host')
+    parser_maintenance.add_argument(
+        '--fsid',
+        help='cluster FSID')
+    parser_maintenance.add_argument(
```

Comment:
we will need to add an option "--check" parameter to the "enter" command. Maybe we will need use separate arguments for each "choice"